hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/TransactionProcessorTest.java

```diff
@@ -447,9 +447,15 @@ private void assertAbortsWith(@NonNull final ResponseCodeEnum reason) {
 
     private void assertAbortsWith(
```

Comment:
Is `assertAbortsWith` still correct?  Consider changing to `assertHaltsWith` since there are no more aborts?

Reply:
By "abort" I mean a `HandleException` prevented us from even entering the EVM (at which point we can either "revert" or "halt").